### ISSUE TITLE
Lag between CMS and app content update

### ISSUE THREAD

**Issue by kulas** [CMP-Studio/cmoa-app-cms#7]
There is considerable lag between the CMS and the app. It appears there is no way to update content without first deleting and then reinstalling the app. This issue has been reported by visitor services.

**Comment by RubenSandwich** [comment]
The app checks for updates on every fresh launch. (Meaning the app is not running in the background when launched. iOS and Android keep apps in the background in order to have app launch faster the next time they are used.) Force quitting the App and then reloading it should be enough to check if new data is needed.

**Comment by kulas** [comment]
Sorry Ruben that is not the case. This is a problem.

**Comment by RubenSandwich** [comment]
I will investigate, but a quick double checking of the code shows me that this is the case. See:
https://github.com/CMP-Studio/cmoa-app-ios/blob/master/International/CIWelcomeViewController.m#L147.

Please provide device specs your seeing this issue on as well as the OS and browser your using when updating the CMS.

**Comment by kulas** [comment]
The devices would be difficult to track as the recent round of complaints are coming from Members. 

My tests: 
Two different iPhone 6 models with 10.2.1 software.
Chrome: Version 55.0.2883.95 (64-bit).

**Comment by RubenSandwich** [comment]
I'm not able to reproduce this issue. Once again the needs to be force quit for the app to resync. 

I pushed some work to resync the app upon every launch, and I tested it on a very simple general case of viewing an artwork that in the sync doesn't exist anymore. But I cannot guarantee that it will work in all cases as the app was only designed to sync at startup. 

I unfortunately don't have anymore time to work on this issue so if that sounds like an acceptable gamble to you then I'll push an update, but if this release causes crashes I won't fix it just pull it out and we will be back to only syncing on every fresh start.
